frontend/src/Dashboard/CrossingUpdatesPage/CrossingUpdates.js

```diff
@@ -12,6 +13,12 @@ class CrossingUpdates extends Component {
   }
 
   render() {
+    const envelope = JSON.parse(this.props.currentUser.communityByCommunityId.viewportgeojson);
```

Comment:
I think this answers my question from above. We store it as a string and use `JSON.parse()` to get it into a format for the map, I suppose?

Reply:
Also, why do we call this envelope? That's a new term to me for this.

Reply:
[ST_Envelope](http://postgis.refractions.net/documentation/manual-1.4/ST_Envelope.html) is a PostGIS bounding box function.

I'm not sure about storing the geojson as text, but calculating it out using PostGIS on read is really rough on performance compared to storing it.